src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -125,6 +126,12 @@ internal class SourcePropertyAccessorSymbol : SourceMemberMethodSymbol
                 syntax,
                 diagnostics);
         }
+
+        private static bool NodeContainsFieldKeyword(CSharpSyntaxNode node)
+            => node.DescendantTokens().Any(
```

Comment:
>DescendantTokens [](http://example.com/codeflow?start=20&length=16)

I am not sure what is going to be more efficient, to go after tokens and then checking their parent, or to go after nodes (```IdentifierNameSyntax```) first and then checking the underlying token.